### ISSUE TITLE
Spike Okta as an alternative to auth0

### ISSUE THREAD

**Issue by digitalronin** [ministryofjustice/cloud-platform#2001]
Is it better/worse?
What's the licensing situation?

**Comment by jasonBirchall** [comment]
https://docs.google.com/document/d/1SJ82-m-ZtI91HcLKJoqAiOPZTvlyQZUHF_ZqaEx6wFk/edit?usp=sharing

TLDR;
In conclusion, Okta offers a very interesting alternative to Auth0. Their intricate licensing structure means we could pay MUCH less for what we need. This would assume any licensing agreements were for this team (Cloud Platform) only. I am aware that other teams in the Ministry of Justice (MoJ) use Auth0 for different purposes, so this would need to be investigated further. 

In Auth0 we currently use GitHub teams to associate a user with a team. The lack of a GitHub social identifier in Okta would mean that we would need to completely change how and what we authenticate against. As stated above, this doesn’t mean GitHub authentication isn’t possible, just that it isn’t easy enough to complete during this sprint.

Due to the wider adoption of Auth0 at the MoJ and the lack of GitHub social identifier in Okta, it is recommended that we persist with Auth0.
